controllers/manager/manager.go

```diff
@@ -49,7 +51,9 @@ const (
 
 const (
 	OAMNetworkType   = "oam"
+	MgmtNetworkType  = "mgmt"
```

Comment:
I would prefer to assign them to the reconciler/controllers, it looks messy in the following switch conditions, can you add a TODO if you are not going to re-position them?